.github/workflows/Renovate.yml

```
@@ -36,7 +36,7 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-    - uses: suzuki-shunsuke/renovate-autoclose-action@befe0dc6bbdea6c28a0125be04496180609b52d2
+    - uses: suzuki-shunsuke/renovate-autoclose-action@2995ce6649b412367776e8f66c5359369a05d2f9
```

Can we pinning this..?

```suggestion
    - uses: suzuki-shunsuke/renovate-autoclose-action@v0.3.2
```